webapp/store/snap_details_views.py

```diff
@@ -17,6 +18,8 @@
 )
 from webapp.markdown import parse_markdown_description
 
+yaml = YAML(typ="safe")
```

Comment:
This can be removed.

Reply:
This is still relevant, no? This file hasn't been changed?